_life/renting_legal.md

```diff
@@ -0,0 +1,73 @@
+- Increase of the IPC: 40% of the relative increase of the index can be applied to your rent: $\Delta R = \frac{\text{new index} - \text{old index}}{\text{old index}} \times 0.4 \times R$
+- Increase of landlord costs: this category is less clearly defined and encapsulates all increases in costs that the landlord can claim.
+
+<p align="center">
+  <img src=rent_increase.jpg/>
+</p>
+
+Above is an example of a rent increase notice, following the official format. By applying the formulas above, one can check that the increases in the first 2 categories indeed follow the law: the mortgage rate increased by 0.5 points, i.e. 2*0.25, hence an increase of  2 * 0.03 * 1210 = 72.60.
+The IPC also increased by about 1%, resulting in an increase of (169.3 - 167.5) / 167.5 * 0.4 * 1210 = 5.2 increase.
+
+The [ASLOCA](/www.asloca.ch)—the Swiss tenant association—provides a calculator on their website that applies the same formulas to check if a rent increase is reasonable or not. If the increase does not follow the formulas above, then you should consider challenging it.
+
+## Rent decreases
+Whenever the indices go down (usually the mortgage rate), you are eligible for a rent decrease. This is your right, and your landlord cannot refuse it if done correctly. The calculator of ASLOCA lets you check how much of a decrease you can ask for, and will provide a [template](https://vaud.asloca.ch/sites/vaud.asloca.ch/files/2025-02/01-lettre-bailleur.docxhttps://vaud.asloca.ch/sites/vaud.asloca.ch/files/2025-02/01-lettre-bailleur.docx)letter to send to the landlord. The decrease will take effect at the next term of your contract, and should be received no later than 3 months prior to that date for the request to be deemed valid.
+
+### Special case: staggered and indexed leases
+Some leases explicitly mention that the rent is *indexed* on the IPC, in which case the rent will increase each year according to only the IPC, regardless of the variation of the mortgage rate. This is generally not a good deal[^indexed], as the IPC rarely decreases, and it means you cannot ask for a rent decrease when the mortgage rate decreases.
+
+Likewise, some leases are staggered: The rent is typically low in the first year, then gradually increases over a 5-year period. In that case, it is also not possible[^staggered] to ask for a rent decrease when the rates go down.
+
+## Challenging a rent increase
+If you deem a rent increase abusive, then you can choose to challenge it. In that case, you must do so within 30 days of receiving the notice, by addressing a letter to the “Commission de conciliation” of your préfecture. ASLOCA also provides a template letter. Note that for the letter to be valid, it should be signed by all the signatories of the lease, including your guarantor if you have one.
```

Comment:
Wouldn't the first step be to talk to your landlord before trying an official challenge? :sweat_smile:

Reply:
Maybe it is worth mentioning at some point that starting any official process will break goodwill with your landlord and it may be better to at least try to resolve it informally beforehand.

Reply:
IMHO that depends on whether your landlord is a person or a corporation. Your goodwill with Bernard Nicod (or other such large corporations) is permanently frozen at zero, so you don't have to worry about breaking it :-)

Reply:
lol good point